### ISSUE TITLE
Add a free field to kafka-connect configs UI

### ISSUE THREAD

**Issue by raphaelauv** [tchiotludo/akhq#705]
My use case is :

I have some specific value.converter.*** or key.converter.*** configurations

- Its not possible to add a non referenced config to the connector from the UI
- configs UI do not show custom config of the current connector, so my previous configuration do not show up

**Comment by tchiotludo** [comment]
Seems to be a good solution I think, there is some edge case for connect configurations (specially custom one). 
PR are welcome :smile:

**Comment by philipp94831** [comment]
Hi, probably related to this issue:
We also use these converter specific configurations. If we now change, e.g., `max.tasks`, using the AKHQ UI, the now updated connector does not include the converter specific configurations anymore. Should I open a new issue for that?

**Comment by tchiotludo** [comment]
I think it's the same issue

**Comment by raphaelauv** [comment]
Yes, I renamed the issue to explicit the support of all current configuration fields of the connector
